tests/rt_threadpool.rs

```diff
@@ -182,3 +182,29 @@ fn idle_after_block_on() {
     rt.shutdown_on_idle();
     assert!(ran.load(Ordering::SeqCst));
 }
+
+#[test]
+fn enter_exposed() {
+    let rt = runtime::Runtime::new().unwrap();
+    rt.enter(|| {
+        let _handle = tokio_02::runtime::Handle::current();
+    });
+}
+
+#[test]
+fn enter_can_spawn_01_futures() {
```

Comment:
Same comments as my suggestions on the `current_thread` version of this test. Additionally, I think we might want to consider having a test where a threadpool runtime's `Handle` is entered on _another_ thread, while the runtime is driven on the main thread, and we spawn a future from inside the `enter` and assert that it runs?

Reply:
OK, I have both tests reliably working right now.

Reply:
🎉